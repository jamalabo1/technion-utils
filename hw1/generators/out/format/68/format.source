B X 702 1
B Y 854 2
B Z 774 3
D W 114 4
A Z 977 5
C Y 876 6
A Y 396 7
C Y 733 8
A Z 620 9
D Z 891 10
D Y 972 11
C Y 153 12
D W 510 13
D Z 349 14
D Y 657 15
D X 315 16
B W 412 17
A W 431 18
C Z 282 19
A Y 764 20
B Y 413 21
A Y 115 22
C X 834 23
C Y 972 24
D Y 441 25
B X 653 26
D W 524 27
D W 811 28
B W 811 29
A Y 420 30
C X 243 31
C W 610 32
B Y 439 33
C Y 229 34
C W 387 35
B W 381 36
B Z 784 37
C X 918 38
B X 8 39
A Z 750 40
C Z 436 41
D W 73 42
D Z 489 43
B Z 393 44
B Z 82 45
B W 934 46
C Z 335 47
D X 726 48
B X 913 49
A Y 45 50
D X 896 51
B X 695 52
A W 949 53
C W 115 54
B Y 241 55
C Z 335 56
A X 301 57
C Z 198 58
D Z 96 59
D W 812 60
C W 941 61
D X 941 62
B Z 30 63
C Y 258 64
B Y 159 65
A Z 485 66
D Z 193 67
A X 774 68
D Z 932 69
B Y 856 70
C X 673 71
C Z 298 72
A Z 92 73
B Y 86 74
D Z 458 75
A Z 61 76
D X 210 77
D W 27 78
B W 145 79
D Y 477 80
D Y 458 81
C Z 758 82
B Y 408 83
A Z 232 84
B Y 563 85
B Z 421 86
C Z 115 87
B X 732 88
B Y 371 89
B W 182 90
B Z 792 91
B W 775 92
B W 457 93
B Y 701 94
B X 774 95
B Z 353 96
A Z 319 97
D W 560 98
A Z 382 99
D Z 656 100
D Y 808 101
A X 546 102
C Y 812 103
D Z 297 104
D X 41 105
A X 339 106
C X 608 107
D Z 73 108
A W 659 109
C X 11 110
C Z 611 111
C Z 445 112
A Y 532 113
B Z 653 114
A X 195 115
C Y 171 116
A Y 932 117
C X 734 118
D Z 518 119
D Z 867 120
C Y 115 121
C W 1 122
A W 48 123
C Y 662 124
D W 834 125
B Y 277 126
D Y 887 127
D X 461 128
C X 714 129
D W 203 130
C Z 13 131
B Y 314 132
C W 399 133
D Y 127 134
A X 760 135
B W 158 136
B Y 342 137
B X 725 138
C Z 913 139
C Y 739 140
B Z 432 141
C Z 631 142
C Y 534 143
A X 546 144
B W 877 145
D Y 790 146
D X 201 147
D Y 751 148
C Z 888 149
C X 305 150
B Z 518 151
D Z 261 152
C Y 977 153
A Y 188 154
A Y 128 155
C Y 598 156
A Z 780 157
C Z 100 158
C Z 523 159
B Z 166 160
B Z 923 161
A W 849 162
D W 654 163
B X 960 164
D X 490 165
B X 774 166
C Z 266 167
C Y 61 168
A X 665 169
A X 918 170
B X 792 171
D Y 663 172
A X 686 173
A X 55 174
C W 504 175