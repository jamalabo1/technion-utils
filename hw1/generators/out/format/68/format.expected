Blockchain Info:
1
Sender Name: C
Receiver Name: W
Transaction Value: 504
Transaction timestamp: 175
2
Sender Name: A
Receiver Name: X
Transaction Value: 55
Transaction timestamp: 174
3
Sender Name: A
Receiver Name: X
Transaction Value: 686
Transaction timestamp: 173
4
Sender Name: D
Receiver Name: Y
Transaction Value: 663
Transaction timestamp: 172
5
Sender Name: B
Receiver Name: X
Transaction Value: 792
Transaction timestamp: 171
6
Sender Name: A
Receiver Name: X
Transaction Value: 918
Transaction timestamp: 170
7
Sender Name: A
Receiver Name: X
Transaction Value: 665
Transaction timestamp: 169
8
Sender Name: C
Receiver Name: Y
Transaction Value: 61
Transaction timestamp: 168
9
Sender Name: C
Receiver Name: Z
Transaction Value: 266
Transaction timestamp: 167
10
Sender Name: B
Receiver Name: X
Transaction Value: 774
Transaction timestamp: 166
11
Sender Name: D
Receiver Name: X
Transaction Value: 490
Transaction timestamp: 165
12
Sender Name: B
Receiver Name: X
Transaction Value: 960
Transaction timestamp: 164
13
Sender Name: D
Receiver Name: W
Transaction Value: 654
Transaction timestamp: 163
14
Sender Name: A
Receiver Name: W
Transaction Value: 849
Transaction timestamp: 162
15
Sender Name: B
Receiver Name: Z
Transaction Value: 923
Transaction timestamp: 161
16
Sender Name: B
Receiver Name: Z
Transaction Value: 166
Transaction timestamp: 160
17
Sender Name: C
Receiver Name: Z
Transaction Value: 523
Transaction timestamp: 159
18
Sender Name: C
Receiver Name: Z
Transaction Value: 100
Transaction timestamp: 158
19
Sender Name: A
Receiver Name: Z
Transaction Value: 780
Transaction timestamp: 157
20
Sender Name: C
Receiver Name: Y
Transaction Value: 598
Transaction timestamp: 156
21
Sender Name: A
Receiver Name: Y
Transaction Value: 128
Transaction timestamp: 155
22
Sender Name: A
Receiver Name: Y
Transaction Value: 188
Transaction timestamp: 154
23
Sender Name: C
Receiver Name: Y
Transaction Value: 977
Transaction timestamp: 153
24
Sender Name: D
Receiver Name: Z
Transaction Value: 261
Transaction timestamp: 152
25
Sender Name: B
Receiver Name: Z
Transaction Value: 518
Transaction timestamp: 151
26
Sender Name: C
Receiver Name: X
Transaction Value: 305
Transaction timestamp: 150
27
Sender Name: C
Receiver Name: Z
Transaction Value: 888
Transaction timestamp: 149
28
Sender Name: D
Receiver Name: Y
Transaction Value: 751
Transaction timestamp: 148
29
Sender Name: D
Receiver Name: X
Transaction Value: 201
Transaction timestamp: 147
30
Sender Name: D
Receiver Name: Y
Transaction Value: 790
Transaction timestamp: 146
31
Sender Name: B
Receiver Name: W
Transaction Value: 877
Transaction timestamp: 145
32
Sender Name: A
Receiver Name: X
Transaction Value: 546
Transaction timestamp: 144
33
Sender Name: C
Receiver Name: Y
Transaction Value: 534
Transaction timestamp: 143
34
Sender Name: C
Receiver Name: Z
Transaction Value: 631
Transaction timestamp: 142
35
Sender Name: B
Receiver Name: Z
Transaction Value: 432
Transaction timestamp: 141
36
Sender Name: C
Receiver Name: Y
Transaction Value: 739
Transaction timestamp: 140
37
Sender Name: C
Receiver Name: Z
Transaction Value: 913
Transaction timestamp: 139
38
Sender Name: B
Receiver Name: X
Transaction Value: 725
Transaction timestamp: 138
39
Sender Name: B
Receiver Name: Y
Transaction Value: 342
Transaction timestamp: 137
40
Sender Name: B
Receiver Name: W
Transaction Value: 158
Transaction timestamp: 136
41
Sender Name: A
Receiver Name: X
Transaction Value: 760
Transaction timestamp: 135
42
Sender Name: D
Receiver Name: Y
Transaction Value: 127
Transaction timestamp: 134
43
Sender Name: C
Receiver Name: W
Transaction Value: 399
Transaction timestamp: 133
44
Sender Name: B
Receiver Name: Y
Transaction Value: 314
Transaction timestamp: 132
45
Sender Name: C
Receiver Name: Z
Transaction Value: 13
Transaction timestamp: 131
46
Sender Name: D
Receiver Name: W
Transaction Value: 203
Transaction timestamp: 130
47
Sender Name: C
Receiver Name: X
Transaction Value: 714
Transaction timestamp: 129
48
Sender Name: D
Receiver Name: X
Transaction Value: 461
Transaction timestamp: 128
49
Sender Name: D
Receiver Name: Y
Transaction Value: 887
Transaction timestamp: 127
50
Sender Name: B
Receiver Name: Y
Transaction Value: 277
Transaction timestamp: 126
51
Sender Name: D
Receiver Name: W
Transaction Value: 834
Transaction timestamp: 125
52
Sender Name: C
Receiver Name: Y
Transaction Value: 662
Transaction timestamp: 124
53
Sender Name: A
Receiver Name: W
Transaction Value: 48
Transaction timestamp: 123
54
Sender Name: C
Receiver Name: W
Transaction Value: 1
Transaction timestamp: 122
55
Sender Name: C
Receiver Name: Y
Transaction Value: 115
Transaction timestamp: 121
56
Sender Name: D
Receiver Name: Z
Transaction Value: 867
Transaction timestamp: 120
57
Sender Name: D
Receiver Name: Z
Transaction Value: 518
Transaction timestamp: 119
58
Sender Name: C
Receiver Name: X
Transaction Value: 734
Transaction timestamp: 118
59
Sender Name: A
Receiver Name: Y
Transaction Value: 932
Transaction timestamp: 117
60
Sender Name: C
Receiver Name: Y
Transaction Value: 171
Transaction timestamp: 116
61
Sender Name: A
Receiver Name: X
Transaction Value: 195
Transaction timestamp: 115
62
Sender Name: B
Receiver Name: Z
Transaction Value: 653
Transaction timestamp: 114
63
Sender Name: A
Receiver Name: Y
Transaction Value: 532
Transaction timestamp: 113
64
Sender Name: C
Receiver Name: Z
Transaction Value: 445
Transaction timestamp: 112
65
Sender Name: C
Receiver Name: Z
Transaction Value: 611
Transaction timestamp: 111
66
Sender Name: C
Receiver Name: X
Transaction Value: 11
Transaction timestamp: 110
67
Sender Name: A
Receiver Name: W
Transaction Value: 659
Transaction timestamp: 109
68
Sender Name: D
Receiver Name: Z
Transaction Value: 73
Transaction timestamp: 108
69
Sender Name: C
Receiver Name: X
Transaction Value: 608
Transaction timestamp: 107
70
Sender Name: A
Receiver Name: X
Transaction Value: 339
Transaction timestamp: 106
71
Sender Name: D
Receiver Name: X
Transaction Value: 41
Transaction timestamp: 105
72
Sender Name: D
Receiver Name: Z
Transaction Value: 297
Transaction timestamp: 104
73
Sender Name: C
Receiver Name: Y
Transaction Value: 812
Transaction timestamp: 103
74
Sender Name: A
Receiver Name: X
Transaction Value: 546
Transaction timestamp: 102
75
Sender Name: D
Receiver Name: Y
Transaction Value: 808
Transaction timestamp: 101
76
Sender Name: D
Receiver Name: Z
Transaction Value: 656
Transaction timestamp: 100
77
Sender Name: A
Receiver Name: Z
Transaction Value: 382
Transaction timestamp: 99
78
Sender Name: D
Receiver Name: W
Transaction Value: 560
Transaction timestamp: 98
79
Sender Name: A
Receiver Name: Z
Transaction Value: 319
Transaction timestamp: 97
80
Sender Name: B
Receiver Name: Z
Transaction Value: 353
Transaction timestamp: 96
81
Sender Name: B
Receiver Name: X
Transaction Value: 774
Transaction timestamp: 95
82
Sender Name: B
Receiver Name: Y
Transaction Value: 701
Transaction timestamp: 94
83
Sender Name: B
Receiver Name: W
Transaction Value: 457
Transaction timestamp: 93
84
Sender Name: B
Receiver Name: W
Transaction Value: 775
Transaction timestamp: 92
85
Sender Name: B
Receiver Name: Z
Transaction Value: 792
Transaction timestamp: 91
86
Sender Name: B
Receiver Name: W
Transaction Value: 182
Transaction timestamp: 90
87
Sender Name: B
Receiver Name: Y
Transaction Value: 371
Transaction timestamp: 89
88
Sender Name: B
Receiver Name: X
Transaction Value: 732
Transaction timestamp: 88
89
Sender Name: C
Receiver Name: Z
Transaction Value: 115
Transaction timestamp: 87
90
Sender Name: B
Receiver Name: Z
Transaction Value: 421
Transaction timestamp: 86
91
Sender Name: B
Receiver Name: Y
Transaction Value: 563
Transaction timestamp: 85
92
Sender Name: A
Receiver Name: Z
Transaction Value: 232
Transaction timestamp: 84
93
Sender Name: B
Receiver Name: Y
Transaction Value: 408
Transaction timestamp: 83
94
Sender Name: C
Receiver Name: Z
Transaction Value: 758
Transaction timestamp: 82
95
Sender Name: D
Receiver Name: Y
Transaction Value: 458
Transaction timestamp: 81
96
Sender Name: D
Receiver Name: Y
Transaction Value: 477
Transaction timestamp: 80
97
Sender Name: B
Receiver Name: W
Transaction Value: 145
Transaction timestamp: 79
98
Sender Name: D
Receiver Name: W
Transaction Value: 27
Transaction timestamp: 78
99
Sender Name: D
Receiver Name: X
Transaction Value: 210
Transaction timestamp: 77
100
Sender Name: A
Receiver Name: Z
Transaction Value: 61
Transaction timestamp: 76
101
Sender Name: D
Receiver Name: Z
Transaction Value: 458
Transaction timestamp: 75
102
Sender Name: B
Receiver Name: Y
Transaction Value: 86
Transaction timestamp: 74
103
Sender Name: A
Receiver Name: Z
Transaction Value: 92
Transaction timestamp: 73
104
Sender Name: C
Receiver Name: Z
Transaction Value: 298
Transaction timestamp: 72
105
Sender Name: C
Receiver Name: X
Transaction Value: 673
Transaction timestamp: 71
106
Sender Name: B
Receiver Name: Y
Transaction Value: 856
Transaction timestamp: 70
107
Sender Name: D
Receiver Name: Z
Transaction Value: 932
Transaction timestamp: 69
108
Sender Name: A
Receiver Name: X
Transaction Value: 774
Transaction timestamp: 68
109
Sender Name: D
Receiver Name: Z
Transaction Value: 193
Transaction timestamp: 67
110
Sender Name: A
Receiver Name: Z
Transaction Value: 485
Transaction timestamp: 66
111
Sender Name: B
Receiver Name: Y
Transaction Value: 159
Transaction timestamp: 65
112
Sender Name: C
Receiver Name: Y
Transaction Value: 258
Transaction timestamp: 64
113
Sender Name: B
Receiver Name: Z
Transaction Value: 30
Transaction timestamp: 63
114
Sender Name: D
Receiver Name: X
Transaction Value: 941
Transaction timestamp: 62
115
Sender Name: C
Receiver Name: W
Transaction Value: 941
Transaction timestamp: 61
116
Sender Name: D
Receiver Name: W
Transaction Value: 812
Transaction timestamp: 60
117
Sender Name: D
Receiver Name: Z
Transaction Value: 96
Transaction timestamp: 59
118
Sender Name: C
Receiver Name: Z
Transaction Value: 198
Transaction timestamp: 58
119
Sender Name: A
Receiver Name: X
Transaction Value: 301
Transaction timestamp: 57
120
Sender Name: C
Receiver Name: Z
Transaction Value: 335
Transaction timestamp: 56
121
Sender Name: B
Receiver Name: Y
Transaction Value: 241
Transaction timestamp: 55
122
Sender Name: C
Receiver Name: W
Transaction Value: 115
Transaction timestamp: 54
123
Sender Name: A
Receiver Name: W
Transaction Value: 949
Transaction timestamp: 53
124
Sender Name: B
Receiver Name: X
Transaction Value: 695
Transaction timestamp: 52
125
Sender Name: D
Receiver Name: X
Transaction Value: 896
Transaction timestamp: 51
126
Sender Name: A
Receiver Name: Y
Transaction Value: 45
Transaction timestamp: 50
127
Sender Name: B
Receiver Name: X
Transaction Value: 913
Transaction timestamp: 49
128
Sender Name: D
Receiver Name: X
Transaction Value: 726
Transaction timestamp: 48
129
Sender Name: C
Receiver Name: Z
Transaction Value: 335
Transaction timestamp: 47
130
Sender Name: B
Receiver Name: W
Transaction Value: 934
Transaction timestamp: 46
131
Sender Name: B
Receiver Name: Z
Transaction Value: 82
Transaction timestamp: 45
132
Sender Name: B
Receiver Name: Z
Transaction Value: 393
Transaction timestamp: 44
133
Sender Name: D
Receiver Name: Z
Transaction Value: 489
Transaction timestamp: 43
134
Sender Name: D
Receiver Name: W
Transaction Value: 73
Transaction timestamp: 42
135
Sender Name: C
Receiver Name: Z
Transaction Value: 436
Transaction timestamp: 41
136
Sender Name: A
Receiver Name: Z
Transaction Value: 750
Transaction timestamp: 40
137
Sender Name: B
Receiver Name: X
Transaction Value: 8
Transaction timestamp: 39
138
Sender Name: C
Receiver Name: X
Transaction Value: 918
Transaction timestamp: 38
139
Sender Name: B
Receiver Name: Z
Transaction Value: 784
Transaction timestamp: 37
140
Sender Name: B
Receiver Name: W
Transaction Value: 381
Transaction timestamp: 36
141
Sender Name: C
Receiver Name: W
Transaction Value: 387
Transaction timestamp: 35
142
Sender Name: C
Receiver Name: Y
Transaction Value: 229
Transaction timestamp: 34
143
Sender Name: B
Receiver Name: Y
Transaction Value: 439
Transaction timestamp: 33
144
Sender Name: C
Receiver Name: W
Transaction Value: 610
Transaction timestamp: 32
145
Sender Name: C
Receiver Name: X
Transaction Value: 243
Transaction timestamp: 31
146
Sender Name: A
Receiver Name: Y
Transaction Value: 420
Transaction timestamp: 30
147
Sender Name: B
Receiver Name: W
Transaction Value: 811
Transaction timestamp: 29
148
Sender Name: D
Receiver Name: W
Transaction Value: 811
Transaction timestamp: 28
149
Sender Name: D
Receiver Name: W
Transaction Value: 524
Transaction timestamp: 27
150
Sender Name: B
Receiver Name: X
Transaction Value: 653
Transaction timestamp: 26
151
Sender Name: D
Receiver Name: Y
Transaction Value: 441
Transaction timestamp: 25
152
Sender Name: C
Receiver Name: Y
Transaction Value: 972
Transaction timestamp: 24
153
Sender Name: C
Receiver Name: X
Transaction Value: 834
Transaction timestamp: 23
154
Sender Name: A
Receiver Name: Y
Transaction Value: 115
Transaction timestamp: 22
155
Sender Name: B
Receiver Name: Y
Transaction Value: 413
Transaction timestamp: 21
156
Sender Name: A
Receiver Name: Y
Transaction Value: 764
Transaction timestamp: 20
157
Sender Name: C
Receiver Name: Z
Transaction Value: 282
Transaction timestamp: 19
158
Sender Name: A
Receiver Name: W
Transaction Value: 431
Transaction timestamp: 18
159
Sender Name: B
Receiver Name: W
Transaction Value: 412
Transaction timestamp: 17
160
Sender Name: D
Receiver Name: X
Transaction Value: 315
Transaction timestamp: 16
161
Sender Name: D
Receiver Name: Y
Transaction Value: 657
Transaction timestamp: 15
162
Sender Name: D
Receiver Name: Z
Transaction Value: 349
Transaction timestamp: 14
163
Sender Name: D
Receiver Name: W
Transaction Value: 510
Transaction timestamp: 13
164
Sender Name: C
Receiver Name: Y
Transaction Value: 153
Transaction timestamp: 12
165
Sender Name: D
Receiver Name: Y
Transaction Value: 972
Transaction timestamp: 11
166
Sender Name: D
Receiver Name: Z
Transaction Value: 891
Transaction timestamp: 10
167
Sender Name: A
Receiver Name: Z
Transaction Value: 620
Transaction timestamp: 9
168
Sender Name: C
Receiver Name: Y
Transaction Value: 733
Transaction timestamp: 8
169
Sender Name: A
Receiver Name: Y
Transaction Value: 396
Transaction timestamp: 7
170
Sender Name: C
Receiver Name: Y
Transaction Value: 876
Transaction timestamp: 6
171
Sender Name: A
Receiver Name: Z
Transaction Value: 977
Transaction timestamp: 5
172
Sender Name: D
Receiver Name: W
Transaction Value: 114
Transaction timestamp: 4
173
Sender Name: B
Receiver Name: Z
Transaction Value: 774
Transaction timestamp: 3
174
Sender Name: B
Receiver Name: Y
Transaction Value: 854
Transaction timestamp: 2
175
Sender Name: B
Receiver Name: X
Transaction Value: 702
Transaction timestamp: 1